evennia/contrib/pathfinding/scripts.py

```diff
@@ -0,0 +1,96 @@
+"""
+PathfinderScript
+
+Contrib - Johnny 2018
+
+A batteries-included abstraction for in-MUD navigation.
+
+Upon initialization, the Pathfinder will quietly build a "map" (directional 
+graph) of all of your game's Rooms and Exits, even accounting for one-way 
+travel.
+
+Requires the `networkx` library, not included with Evennia, but only a
+`pip install networkx` away.
+
+---
+
+Installation:
+
+Get or create an instance of this script at server start/reload.
+
+To do so, modify `yourgame/server/conf/at_server_startstop.py`:
+
+Add this import to the top of the file:
+
+    from evennia.contrib.pathfinding.scripts import PathfinderScript
+    
+Add this line to the beginning or end of your `at_server_start()` function, but
+above the word 'pass' if it's there:
```

Comment:
Actually, you replace the `pass`, it has no purpose if you are actually doing anything in the function  ...